### ISSUE TITLE
irsa issue

### ISSUE THREAD

**Issue by toreydai** [aws-samples/eks-workshop-greater-china#8]
In Step 7.3
Before deploy iam-pod，you need to create service account iam-test first.
Below is the command:
eksctl create iamserviceaccount --name iam-test --namespace default \
    --cluster ${CLUSTER_NAME} --attach-policy-arn arn:aws-cn:iam::aws:policy/AmazonS3ReadOnlyAccess \
    --approve --override-existing-serviceaccounts --region ${AWS_REGION}

**Comment by liangruibupt** [comment]
7.1.1 使用eksctl 创建service account 已经创建s3-echoer irsa
7.3 说明中已经告知需要替换
 Please replace the serviceAccountName: s3-echoer

我会更新代码，直接用 china/2020_EKS_Launch_Workshop/resource/IRSA/iam-pod.yaml

**Comment by phoenixyy** [comment]
In Step 7.3. 

You can only add the env AWS_DEFAULT_REGION instead of AWS_REGION. If you add env AWS_REGION only, it will get "no provider issue" as well

**Comment by liangruibupt** [comment]
@phoenixyy Yes, in china/2020_EKS_Launch_Workshop/resource/IRSA/iam-pod.yaml, we use the AWS_DEFAULT_REGION
        env:
        - name: AWS_DEFAULT_REGION
          value: "cn-northwest-1"
